packages/clerk-js/package.json

```diff
@@ -81,14 +85,17 @@
     "swr": "2.3.3"
   },
   "devDependencies": {
+    "@clerk/testing": "workspace:^",
     "@emotion/jest": "^11.13.0",
+    "@playwright/test": "^1",
```

Comment:
isn't this a root workspace dependency? does it need to be installed in the `clerk-js` package.json?